frontend/pages/weights/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { GetServerSideProps, InferGetServerSidePropsType } from "next"
+import Head from "next/head"
+import { Button } from "../../components/Button/Button"
+import { Headline } from "../../components/Headline/Headline"
+import { ItemPreview } from "../../components/Item/ItemPreview"
+
+// As long as we do not have a weight. Let's work with Todo
+type Todo = {
+    userId: number
+    id: number
+    title: string
+    completed: boolean
+}
+
+type WeightsListProps = {
+    items: Todo[],
+    currentPage: number,
+    limit: number
+}
+
+/** Base List for weights */
+export default function WeightsList({ items, currentPage, limit }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+    const siteTitle = `Page ${currentPage} - Discover weights`
```

Comment:
Title is not good. May we update when we have weights. I would go into something like this:

```
Latest - World Wide Weights
Latest | Page 2 - World Wide Weights

2347 Weights - World Wide Weights
2347 Weights - World Wide Weights | Page 2
```